packages/reports/src/sass/app/_common/layout/trader-layouts.scss

```diff
@@ -43,7 +43,6 @@
         .top-widgets-portal {
             position: absolute;
             top: 0px;
-            width: 100%;
```

Comment:
From my stand point, `width: 100%` is not needed for this className (moreover, it caused a bug), so I removed it from css files and from _top-widgets.tsx_ where it was written inline. Added sanity for qa-checklist

Reply:
This might be a problem in browsers like Safari or Firefox. Please check once.

Reply:
Thats screenshots from Safari: test link and production (no difference)
<img width="487" alt="Screenshot 2024-03-12 at 8 45 25 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/066ba2fb-c80f-45d5-8211-33038de6a6b7">
<img width="487" alt="Screenshot 2024-03-12 at 8 45 40 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/80e9f953-e278-4e4b-8a5a-0840149218d7">
<img width="487" alt="Screenshot 2024-03-12 at 8 45 43 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/4d6304c3-5243-4ca5-97fc-43985789cef7">
<img width="487" alt="Screenshot 2024-03-12 at 8 48 07 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/47705ded-2687-4cb9-b6ac-69d5dec728d3">
<img width="487" alt="Screenshot 2024-03-12 at 8 48 16 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/02c6cfa6-d227-4395-85e2-fa254b152a2c">

Reply:
Alright. Thanks for checking

Reply:
Thats screenshots of the test link from Firefox, also same as on production
<img width="1182" alt="Screenshot 2024-03-12 at 8 51 08 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/c883d614-4dbf-4ea1-940c-32c69ccaa130">
<img width="1182" alt="Screenshot 2024-03-12 at 8 51 24 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/978ae615-9582-4481-9ce6-0075e0f6b9ea">
<img width="1182" alt="Screenshot 2024-03-12 at 8 51 28 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/cd55145d-d503-4bf5-a211-9cf2d1e7d4d9">

Reply:
+ I also added check into the qa-checklist